### ISSUE TITLE
ABR test fail

### ISSUE THREAD

**Issue by DanielMartine** [gpac/gpac#1008]
Hi there! This night I put online the MP4Box based media server for live streaming:
http://178.63.175.10/
I can not get the adaptive bit-rate behavior,
the stream is unstable and stops before displaying the correct adaption set.
Check keyframes allignment i get:
`MP4Box -info 0 1080set.mp4 2>&1 | grep GOP`
**Average GOP length: 50 samples**
The command line i use is:
`MP4Box -dash-live 8000 -subdur 8000 -bound -profile dashavc264:live -mpd-refresh 3600.0 -time-shift 24 -min-buffer 4000 -insert-utc -no-cache -out manifest.mpd 1080set.mp4#video 720set.mp4#video 360set.mp4#video 1080set.mp4#audio 720set.mp4#audio 360set.mp4#audio`

May i ask a special help to understand whats going here?
Thanks

**Comment by jeanlf** [comment]
Just tested, seems to work fine on your website and in MP4Client as well no ?

**Comment by DanielMartine** [comment]
Hi, I tried to insert the manifest address in several web tools to probe the operation but I was not able to upload the file:

[YOUTUBE DASH CONFORMANCE TOOL](http://dash-mse-test.appspot.com/dash-player.html?url=http%3A%2F%2F178.63.175.10%2Fvideo%2Fmanifest.mpd&autoplay=on&adapt=auto&flavor=)

[DASH.JS WEB PLAYER](https://reference.dashif.org/dash.js/1.4.0/samples/dash-if-reference-player/index.html) This whould be very good for testing.

Do i need to set CORS on webserver to use the tool above?

The only one who accept my live mpd is :
[ITEC MPD VALIDATOR](http://www-itec.uni-klu.ac.at/dash/?page_id=605#)
Which say:

> If more than one Media Segment is present the duration attribute or SegmentTimeline element shall be present.

In short, I would like to be sure about more aspect i can check, I have no idea how to do whith `-segment-timeline` flag: if i use it i'm unable to display anything due to:

> [DASH] Drift between minCTS of segment and MPD start time is 0.0209375 s

Thanks a lot
Dan

**Comment by jeanlf** [comment]
yes you need to setup CORS to use your manifest in dash.js ref soft or other players
SegmentTimeline should work in live, we'll investigate if there's an issue here

**Comment by DanielMartine** [comment]
Do you want that i close this and open one another issue for `-segment-timeline` in live? 
If you need some beta tests just ask.
Thanks for your work!
